### ISSUE TITLE
What's the different between `tiflow/cdc/api` and `tiflow/pkg/api`?

### ISSUE THREAD

**Issue by lonng** [pingcap/tiflow#6036]
### Before asking a question, make sure you have

- [X] Searched existing Stack Overflow questions
- [X] Googled your question
- [X] Searched open and closed [GitHub issues](https://github.com/pingcap/tiflow/issues?q=is%3Aissue)
- [X] Read the [documentation](https://docs.pingcap.com/tidb/stable)

### What is your question?

Seems that the two packages are used to implement some parts of TiCDC OpenAPI.

**Comment by lonng** [comment]
@asddongmen Can we merge the two packages?

**Comment by zhaoxinyu** [comment]
@lonng,`tiflow/cdc/api` provides the OpenAPI implementation, while `tiflow/pkg/api` supplied with a rest client is used to interacting with the OpenAPI. At present, `tiflow/pkg/api` is mainly used by cli commands. In the future, it can be extracted to another standalone project such as 'tiflow-client-go'. What's your opinion? @asddongmen @sdojjy

**Comment by asddongmen** [comment]
> @lonng,`tiflow/cdc/api` provides the OpenAPI implementation, while `tiflow/pkg/api` accompanied with a rest client is used to interacting with the OpenAPI. At present, `tiflow/pkg/api` is mainly used by cli commands. In the future, it can be extracted to another standalone project such as 'tiflow-client-go'. What's your opinion? @asddongmen @sdojjy

Yes, I believe that we can merge these two packages in the near future.
